sendgrid-email/addtolist_es6.js

```diff
@@ -4,8 +4,8 @@ module.exports = (event) => {
 
   return new Promise((resolve, reject) => {
     var request = sg.emptyRequest()
-    const { email, first_name: nameFirst, last_name: nameLast } = event.data.User.node
-    request.body = [{email, first_name, last_name }];
+    const { email, nameFirst, nameLast } = event.data.User.node
+    request.body = [{email, first_name: nameFirst, last_name: nameLast }];
```

Comment:
Hi @kbrandwijk. The syntax `{ email, first_name, last_name } = event.data.User.node` doesn’t make sense if the schema is `nameFirst`, `nameLast`, etc, no?